ovos_dinkum_listener/voice_loop/hotwords.py

```diff
@@ -71,13 +71,15 @@ class HotwordState(str, Enum):
 class HotwordContainer:
     _plugins = {}
 
-    def __init__(self, bus=FakeBus(), expected_duration=3, sample_rate=16000, sample_width=2):
+    def __init__(self, bus=FakeBus(), expected_duration=3,
+                 sample_rate=16000, sample_width=2):
         self.bus = bus
         self.state = HotwordState.HOTWORD
         # used for old style non-streaming wakeword (deprecated)
         self.audio_buffer = CyclicAudioBuffer(expected_duration,
                                               sample_rate=sample_rate,
                                               sample_width=sample_width)
+        self.applied_hotwords_config = None
```

Comment:
can  this be shorter? just `hotwords_config` ?

Reply:
This will probably be removed; hotwords reference other configuration values, so they should probably just reload on any change to ensure they're up to date (i.e. if the default language is changed, all hotwords should be reloaded